enarxbot-pr-assign

```diff
@@ -85,10 +85,12 @@ def iterusers(nested_dictionary):
 
     for key, value in nested_dictionary.items():
         if key in ["author", "who"]:
-            yield value
+            if value:
```

Comment:
I suspect you probably want `if value is not None:` in both cases.

Reply:
@npmccallum We _do_ get a value in both cases -- not `None`. It's just an empty dictionary.

Reply:
@npmccallum Here's an example printout of `value` from a run locally:

```
{'login': 'ambaxter', 'id': 'MDQ6VXNlcjE1MDA2Nzgw'}
{'login': 'npmccallum', 'id': 'MDQ6VXNlcjI4ODMwNA=='}
{'login': 'haraldh', 'id': 'MDQ6VXNlcjQyNzgyNQ=='}
{}
{'login': 'ueno', 'id': 'MDQ6VXNlcjQ3NTQ5'}

```

The empty dictionary is what's causing the problem.

Reply:
Then check for `len(value) == 0`.

Reply:
Fair enough. Amended.